sdcm/tester.py

```diff
@@ -2234,7 +2234,7 @@ def create_materialized_view(self, ks_name, base_table_name, mv_name, mv_partiti
         self.log.debug('MV create statement: {}'.format(query))
         session.execute(query)
 
-    def _wait_for_view(self, scylla_cluster, session, key_space, view):
+    def _wait_for_view(self, scylla_cluster, session, key_space, view, wait_attempts=20):
```

Comment:
How do you know that 20 attempts is long enough?

Reply:
20 attemts is hardcoded in the code:
https://github.com/scylladb/scylla-cluster-tests/blob/892a54f2e802c9189be1cdf55cf18d2f4d2f84d4/sdcm/tester.py#L2257

I change it to be a flexible